src/vid2805-dual-stepper-element.ts

```diff
@@ -0,0 +1,191 @@
+import { html, LitElement, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin } from '.';
+import { mmToPix } from './utils/units';
+
+export type HandShape = 'arrow' | 'plain'; // | 'fancy';
```

Comment:
Remove 'fancy'

Reply:
this was a placeholder for an ornate hand paths that I was in the process of adding. Now added